pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -422,16 +425,6 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 		return reconcile.Result{}, err
 	}
 
-	esgwCertificate, err := certificateManager.GetCertificate(r.client, relasticsearch.PublicCertSecret, common.OperatorNamespace())
```

Comment:
From what I can tell, intrusion-detection-controller is one of the few managed cluster components that still talks directly to elasticsearch via es-gateway. 

It does talk to Linseed in most codepaths, but when searching for security events it seems it still talks to ES.